experimental/examples/chat-mpp/shared/src/iosMain/kotlin/main.ios.kt

```diff
@@ -3,6 +3,13 @@ import platform.UIKit.UIViewController
 
 fun MainViewController(): UIViewController =
```

Comment:
May be we should add other than Application method for  wrapping Compose to UIViewController.
And it is not MainViewController  now  but just ChatWindow

Reply:
I will try to do it

Reply:
And please update README.md telling about interop on iOS when you finish with it

Reply:
Done.
Found an easier way to correctly initialize Compose ViewController size at startup with SwiftUI. 
```Kotlin
ComposeViewControllerToSwiftUI()
    .ignoresSafeArea(.keyboard)
```
ComposeView will be initialized with bounds between top and bottom SwfitUI edges. And size will not be changed on showing keyboard.